ckanext/geocat/utils/csw_mapping.py

```diff
@@ -128,11 +146,19 @@ def get_metadata(self, csw_record_as_string, geocat_id):
                                    organization_slug=self.organization_slug,
                                    valid_identifiers=self.valid_identifiers)
         dataset_dict['owner_org'] = self.organization_slug
+
         rights = \
             _map_dataset_rights(node=root_node,
                                 terms_of_use=self.terms_of_use_graph,
                                 default_rights=self.default_rights)
 
+        # set license to rights
+        # if rights name is found in the license vocabulary
+        license = rights
```

Comment:
You're setting `license = rights` twice here - I think this line is a mistake.

Reply:
oh yes, you are right, the check already exists.